src/ophys_etl/transforms/suite2p_container.def

```diff
@@ -0,0 +1,44 @@
+Bootstrap: docker
+From: continuumio/miniconda3:4.8.2
+
+%post
+    # these can be set during build like
+    # sudo SINGULARITYENV_SUITE2P_TAG=0.7.5 singularity build ...
+    SUITE2P_TAG=${SUITE2P_TAG:-0.7.5}
+    OPHYS_ETL_TAG=${OPHYS_ETL_TAG:-master}
+
+    export PATH=/opt/conda/bin:${PATH}
+
+    # initialize conda
+    CONDA_BASE=$(conda info --base)
+    . $CONDA_BASE/etc/profile.d/conda.sh
+
+    # make this variable available at runtime
+    echo "export CONDA_BASE=\"${CONDA_BASE}\"" >> $SINGULARITY_ENVIRONMENT
+
+    # install Suite2P
+    apt-get install -q -y libgl1-mesa-glx
+    git clone https://github.com/MouseLand/suite2p
+    cd suite2p
+    git checkout ${SUITE2P_TAG}
+    conda env create -q -f environment.yml
+    conda activate suite2p
+    pip install -q .
```

Comment:
`pip install --no-cache-dir -q .`

Should be an easy change that should shrink the image immediately.

Reply:
done, reduced by ~70M.